crates/edr_test_utils/Cargo.toml

```diff
@@ -6,6 +6,14 @@ edition.workspace = true
 [dependencies]
 serde.workspace = true
 serde_json.workspace = true
+foundry-common.workspace = true
+foundry-compilers = { workspace = true, features = ["project-util"] }
+
+fd-lock = "4.0.0"
+once_cell = "1"
+regex = "1"
+tracing-subscriber = { workspace = true, features = ["env-filter", "fmt"] }
+rand.workspace = true
```

Comment:
```suggestion
rand.workspace = true
regex = "1"
tracing-subscriber = { workspace = true, features = ["env-filter", "fmt"] }
```

alphabetic ordering

Reply:
Thanks, fixed in [32ab634](https://github.com/NomicFoundation/edr/pull/619/commits/32ab634bfb021e5404c0bbb63c096935ebc28900)